src/components/slider/examples/slider-multiplier-percentage-colors.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * With percentage colors
+ * You can add the `displays-percentage-colors` class to your slider component
+ * and it will automatically visualize current percentage colors in real-time.
+ *
+ * The colors change with intervals of 10 as users drags the slider pin.
+ * The color spectrum is not modifieable, and looks like red → orange → yellow
```

Comment:
Since the colours are defined on `:host` in `percentage-color.scss`, I'm pretty sure they can actually be modified… but maybe we don't want to document that in case we want to change the implementation?

Reply:
I don't know. I think @Kiarokh said that we would have to write the css a little bit different so that it can be modified.

Reply:
Hmm… 🤔

Regardless, it's probably best to claim it's not possible to modify to begin with, then if we get requests to make it possible to modify, we can consider if we just wish to expose the ten steps we currently have, or make the steps modifiable too, and go from there, without potentially having to make a breaking change 👍

Reply:
I can easily imagine that some people want other color spectrums, e.g. from cold to hot (blue to red) or even the same spectrum we have but in reverse. 

It's super easy to make the colors externally modifiable. It's only some CSS work and a bit of additional documentation.

However, it will be only for intervals of 10. So they can't for example switch to another color when they reach 35, and to another color at 75.

If people ask for it, we can easily add it. (or we just add this possibility in future ourselves)

Reply:
It's already possible:

![image](https://user-images.githubusercontent.com/154725/108178349-a2332200-7104-11eb-91dc-84bde2d96de8.png)

But yes, the variable has to be defined right on the `limel-slider` element, and by changing the CSS a little we can make it possible to override the colours by defining the variables on any ancestor element.

Regardless, let's keep that and the documentation as an easy feature to add in the future 🙂